### ISSUE TITLE
Page could be loaded because of "net::ERR_CACHE_MISS"

### ISSUE THREAD

**Issue by leiweibo** [gree/unity-webview#71]
I tried to load a live url as following code: 

> webviewObject = (new GameObject("WebViewObject")).AddComponent<WebViewObject>();
>         webviewObject.Init();
> 
>         webviewObject.SetMargins(5, 5, 5, Screen.height / 4);
>         webviewObject.SetVisibility(true);
>         webviewObject.LoadURL("http://www.unity3d.com");

but, I get error msg in my browser:

"The webpage at http://www.unity3d.com/ could not be loaded because: 
net::ERR_CACHE_MISS

I also tried remove the following in the SampleScript:

>       var src = System.IO.Path.Combine(Application.streamingAssetsPath, Url);
>       var dst = System.IO.Path.Combine(Application.persistentDataPath, Url);
>       var result = "";
>       if (src.Contains("://")) {
>         var www = new WWW(src);
>         yield return www;
>         result = www.text;
>       } else {
>         result = System.IO.File.ReadAllText(src);
>       }
>       System.IO.File.WriteAllText(dst, result);

and use webviewObject.LoadURL("http://www.unity3d.com"); that try to load the live directly.
I get the same error. But this work fine on iOS.

If I don't remove above code, everything work fine.

(Ps: Android version. 6.0)

**Comment by KojiNakamaru** [comment]
I've mofidied SampleWebView.cs to allow ordinal http/https Url in https://github.com/gree/unity-webview/commit/770f34b6b5253bfa2e25fb651e72c62834ebe03c and have confirmed that it worked for http://www.unity3d.com/ on Nexus 7 (2013, Android 6.0.1). 

The `net::ERR_CACHE_MISS` might be caused if the internet access was disabled so could you please check `Internet Access` in the project settings for Android. Please also follow https://github.com/gree/unity-webview#android to update AndroidManifest.xml adequately for better performance.

**Comment by leiweibo** [comment]
Thanks! the ordinal http/https URL works fine now.

About ERR_CACHE_MISS, I spent lots of my time to investigate it yesterday, I found if I use 
> IEnumerator Start()
> ....
> Www www = new Www(url); 
> yeild return www;

it worked fine, but it doesn't work when I only use 
> void Start();

But I don't know why.

Your comment remind me, than I try change `Internet Access` from `auto` to `required`.
Everything works fine now.

Thank you again, man!
